content/references/protocol-reference/transactions/transaction-types/nftokenacceptoffer.md

```diff
@@ -80,7 +80,9 @@ If both offers are for the same asset, it is possible that the order in which fu
 
 In brokered mode, The offers referenced by `BuyOffer` and `SellOffer` must both specify the same `TokenID`; that is, both must be for the same `NFToken`.
 
+## Error Cases
 
+* If there is no entry with the given `SellOffer` or `BuyOffer` index on the ledger, the transaction fails with the result `tecOBJECT_NOT_FOUND`.
```

Comment:
I suspect there are other error cases not noted here.

Also, please add the new error code to https://github.com/XRPLF/xrpl-dev-portal/blob/master/content/references/protocol-reference/transactions/transaction-results/tec-codes.md